react/features/base/react/components/native/styles.js

```diff
@@ -9,13 +9,27 @@ const HEADER_HEIGHT = 44;
 const HEADER_PADDING = BoxModel.padding;
 
 export const STATUSBAR_COLOR = ColorPalette.blueHighlight;
+export const SIDEBAR_WIDTH = 250;
 
 /**
  * The styles of the React {@code Components} of the generic components
  * in the app.
  */
 export default createStyleSheet({
 
+    /**
+     * The topmost container of the animated side bar.
+     */
+    animatedMenuContainer: {
+        bottom: 0,
+        flexDirection: 'row',
+        left: -SIDEBAR_WIDTH,
+        position: 'absolute',
+        top: 0,
+        width: SIDEBAR_WIDTH,
+        zIndex: 999999
```

Comment:
why do we need such zindex? can we make sure this is the topmost component in the welcome page hierarchy instead?